package.json

```diff
@@ -3,6 +3,11 @@
 	"version": "0.0.1",
 	"license": "UNLICENSED",
 	"private": true,
+	"engines": {
```

Comment:
Please use ">=" instead of "=" so that updated versions can still be used

Reply:
See my comment on the core